.eslintignore

```diff
@@ -1,5 +1,5 @@
 node_modules/**
 /client/**
-/modules/library/**
+/modules/ponyfill/**
```

Comment:
ugh, this is such a dumb and meaningless name

Reply:
@ljharb I agree, but users too often use it. That could be clearer for understanding functionality. However, this renaming is not the final decision, now it's just an experiment how it can look.

Reply:
My suggestion for definitions is established [here](https://github.com/es-shims/es-shim-api).

Regardless of what you pick, I'd strongly suggest not endorsing a horrible made-up name that has no implicit etymological meaning.

Reply:
`polyfill` will confuse users even more. My opinion about this case - the word `ponyfill` is something horrific, but I don't see any other words (not phrases from several words) which could describe it for almost all - it's enough popular (https://github.com/sindresorhus/ponyfill, https://ponyfoo.com/articles/polyfills-or-ponyfills, etc). Let's try to find any alternative -)